lib/features/steps/multipart_request_steps.rb

```diff
@@ -18,28 +18,77 @@ def valid_multipart_form_data?(request)
   assert(request[:body].size.positive?, "Multipart request payload contained #{request[:body].size} fields")
 end
 
+# Verifies that any type of request contains multipart form-data
+#
+# @step_input request_type [String] The type of request (error, session, build, etc)
+Then('the {word} request is valid multipart form-data') do |request_type|
+  list = Maze::Server.list_for request_type
+  valid_multipart_form_data?(list.current)
+end
+
+# (Deprecated) Retained for backwards compatibility
+# Use "the {word} request is valid multipart form-data"
+#
 # Verifies that the request contains multipart form-data
 Then('the request is valid multipart form-data') do
-  valid_multipart_form_data?(Maze::Server.errors.current)
+  $logger.warn 'This step is deprecated and may be removed in a future release'
```

Comment:
Could we do something like raise a ticket under a v6 epic to ensure we do remove this at the next major?